db/migrate/20190925162318_add_serial_normalized_to_external_registry_bike.rb

```diff
@@ -0,0 +1,9 @@
+class AddSerialNormalizedToExternalRegistryBike < ActiveRecord::Migration
+  def change
```

Comment:
please merge this into the create migration, since they're occurring in the same PR

Reply:
Squashed in 3fce2c5c